src/data_structures/graph.ml

```diff
@@ -79,4 +136,367 @@ let is_subgraph graph subgraph =
   compare_nodes (Array.to_list graph.nodes) (Array.to_list subgraph.nodes)
   && res
 
-let get_info graph = (Array.get graph.nodes 1).info (* just to use info *)
+module IntPair = struct
```

Comment:
Could you add a comment saying this should be replaced by the brand new [`Pair`](https://github.com/ocaml/ocaml/blob/trunk/stdlib/pair.mli#L18) module in 5.4?